lib/puppetlabs_spec_helper/version.rb

```diff
@@ -1,5 +1,3 @@
 module PuppetlabsSpecHelper
-  module Version
-    STRING = '1.0.1'
-  end
+  VERSION = '1.0.1'
```

Comment:
Could we leave PuppetlabsSpecHelper::Version::STRING just in case someone is using it?